src/containers/simulator/simulatorContainer.js

```diff
@@ -15,14 +15,16 @@ import {
   step,
   run,
   pause,
-  getCoreInstructions
+  getCoreInstructions,
+  incrementProcessRate,
+  decrementProcessRate
 } from '../../modules/simulator'
 
 import {
   removeWarrior
 } from '../../modules/parser'
 
-const SimulatorContainer = ({ isRunning, isInitialised, coreSize, instructions, parseResults, roundResult, init, step, run, pause, getCoreInstructions, removeWarrior }) => (
+const SimulatorContainer = ({ isRunning, isInitialised, coreSize, instructions, parseResults, roundResult, init, step, run, pause, getCoreInstructions, removeWarrior, incrementProcessRate, decrementProcessRate }) => (
```

Comment:
Need to find a way to do this better. Code is griping if I split the lines for some reason. Not sure if it's a genuine problem or just some weird syntax highlighting quirk!